### ISSUE TITLE
Overview fires imageloadstart multiple times and bug main map

### ISSUE THREAD

**Issue by kabog** [openlayers/openlayers#9796]
Dup of : 

https://github.com/openlayers/openlayers/issues/8513
https://github.com/openlayers/openlayers/issues/8290
https://github.com/openlayers/openlayers/issues/8728

But all closed with no resolution (I think)

**Comment by mike-000** [comment]
In OpenLayers 6 layers can only be used in a single map https://github.com/openlayers/openlayers/blob/master/changelog/upgrade-notes.md#layers-can-only-be-added-to-a-single-map but if layers share the same source the problem is still there as can be seen when the #8728 fiddle is updated to replace v4.6.5 with the master full build http://jsfiddle.net/ykh692ne/3/  But the workaround suggested there still applies - use separate sources as well as layers


```
var sourceOptions = {
        url: 'http://maps.dwd.de/geoserver/dwd/wms',
        params: {
          LAYERS: 'dwd:bluemarble',
          TRANSPARENT: true,
          VERSION: '1.3.0'
        }
      };

var ovMap = new ol.control.OverviewMap(
  {
    layers: [
      new ol.layer.Image({
        source: new ol.source.ImageWMS(sourceOptions)
      })
    ],
    collapsed: true,
    collapsible: true
  }
);

var map = new ol.Map({
  layers: [
    new ol.layer.Image({
      source: new ol.source.ImageWMS(sourceOptions)
    })
  ],
  controls: [ovMap],
```


Also rastersources cannot be used twice - see https://stackoverflow.com/questions/56992171/multiple-imagelayer-with-rastersource-not-visible  Ideally such limitations should be mentioned in the API.

**Comment by ahocevar** [comment]
What makes you think raster sources cannot be used by more than one layer? See https://codesandbox.io/s/simple-r0bdx that shows an OSM source used by two layers in two maps.

So maybe this issue is specific to `ol/OverviewMap`? If anyone wants to fix that, we could use a maintainer of the OverviewMap. If issues keep popping up around that, we'll have to remove it from the library.

**Comment by mike-000** [comment]
> What makes you think raster sources cannot be used by more than one layer? See https://codesandbox.io/s/simple-r0bdx that shows an OSM source used by two layers in two maps.
> 
> **EDIT:** Oh, it seems you meant `ol/source/Raster`. That's different than a tile image source. A documentation improvement pull request would be welcome.
> 
> ~So maybe this issue is specific to `ol/OverviewMap`? If anyone wants to fix that, we could use a maintainer of the OverviewMap. If issues keep popping up around that, we'll have to remove it from the library.~

Yes, and ImageWMS sources can also be shared without problems (I've tested that with maps sharing a view when the map sizes differ, and views in different projections when the ImageWMS projection is defaulted) so it does seem to be an issue with OverviewMap and main map synchronisation.

#8513 involved ol/source/Raster so it was unlikely to work regardless of what is causing the problem with ol/source/ImageWMS.

**Comment by mike-000** [comment]
I suspect what is happening is similar to the problem in https://stackoverflow.com/questions/56090823/synchronize-different-maps-with-different-center-and-rotation which meant that simple setCenter(), setRotation() and setResolution() calls needed to be replaced by animate().  In OL6 the callbacks and flags would be unnecessary as calling getAnimating() works equally well to check if the other view is animating.  OverviewMap.js uses setCenter(), setRotation() and fit().  The first two could be replaced by animate(), and fit() would need an explicit duration (which could be zero).

**Comment by mike-000** [comment]
I have identified the problem in earlier versions of OpenLayers as the `ovview.fit(extent)` call which doesn't set the desired extent, resulting in an endless loop of attempts.  It is easily fixed by changing to `ovview.fit(extent, {constrainResolution: false})`; (tested using a patched 4.6.5 debug full build).  When using the #8728 jsfiddle with an OL6 full build a different loop is caused by the parent map view having an extent constraint (demonstrated by removing the constraint).  Potentially any of the `setCenter`, `setRotation` or `fit` calls in OverviewMap.js which conflicts with view constraints (including the default `multiWorld: false`) could now cause problems.
